### PR TITLE
Bugfix/onion horoscope change

### PR DESCRIPTION
**Pull request by Zvax**
reviving somewhat the horoscope plugin

removed the date range as I can't extract it from the page anymore

cheers

### CODE REVIEW

**Comment by Zvax** [comment]
I'm not quite sure about the xpath commands, I'm new to this, so don't hesitate to point out flaws please.

File: src/Plugins/Horoscope.php [hunk 1]
```diff
@@ -57,23 +72,55 @@ public function divine(Command $command)
...
+                $currentHoroscopeUrl
             )
         );
     }
 
+    private function extractCurrentHoroscopeUrl(\DOMXPath $xpath): string
+    {
+        $url = $xpath->evaluate("
+            string(
+                //a[contains(@class,'js_entry-link')][1]
```
Comment (PeeHaa) [inline src/Plugins/Horoscope.php]: When targeting classes it's generally better to normalize and exact match.  
Your current solution also matches classes like `foojs_entry-linkbar`.

What you want to do is something like `//a[contains(concat(" ", normalize-space(@class), " "), " js_entry-link ")][1]`

Comment (PeeHaa) [reply src/Plugins/Horoscope.php]: Oh it looks like you refactored this piece out already.

**Comment by Zvax** [comment]
ok, I realize "don't match anything" is not the same as don't match everything, but yeah. should not match everything now :)

**Pull request merged**